patient_case_summary/patient_case_summary.py

```
@@ -0,0 +1,788 @@
+# This tutorial shows you how to build an agentic workflow that can extract key details from a given patient, check clinical guidelines to see whether the patient's status meets recommended care standards, and then produce a human-readable case summary that a clinician can review.
+# 
+# ![](patient_case_summary.png)
+# 
+# We use [Synthea](https://github.com/synthetichealth/synthea) to generate synthetic patient data, which contains a comprehensive set of information surrounding the patient. This includes, among other attributes, active patient conditions, encounters, and medications.
+# 
+# Given a sample patient, we extract out the relevant information, and use it to retrieve relevant medical guidelines that provide recommendations on continued treatment, and present it to a clinician.
+
+# In[ ]:
+
+
+get_ipython().system('pip install llama-index llama-index-indices-managed-llama-cloud llama-cloud llama-parse')
```

Ruff (0.8.2)

Undefined name `get_ipython`

(F821)